### ISSUE TITLE
It done broke

### ISSUE THREAD

**Issue by adubry75** [JulienMelissas/wtfwl#8]
see title.

**Comment by JulienMelissas** [comment]
Yeah... it's fucking sad. Apple bought Dark Sky and discontinued their API. I haven't been able to find a weather API that has similar features at a low enough price point. Will try and take a look again over the holiday season to see if there are some good options!
